README.md

```diff
@@ -95,6 +97,8 @@ Sends messages to the specified queue using batch API to reduce number of calls.
     - **body** - { Any } - the body of the message
     - **[delaySeconds]** - { Number } - the length of time, in seconds, for which a specific message is delayed
     - **[messageAttributes]** - { Object } - key-value pairs of attributes to be added to the message
+    - **[dedupId]** - { String } - Required for and only used in FIFO queues: https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/using-messagededuplicationid-property.html
+    - **[groupId]** - { String } - Required for and only used in FIFO queues: https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/using-messagegroupid-property.html
```

Comment:
1. lower case `required`
2. please, hide links like "required for and only used in FIFO queues ([docs](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/using-messagededuplicationid-property.html))"